.mention-bot

```diff
@@ -0,0 +1,4 @@
+{
+  "requiredOrgs": ["theforeman"]
+  "userBlackList": ["abenari", "unorthodoxgeek", "alongoldboim", "bgupta", "logicminds"]
```

Comment:
Shouldn't we "remove" this guys from the organization if they're definitely inactive?